bridgetown-core/lib/bridgetown-core.rb

```diff
@@ -111,6 +110,16 @@ module Bridgetown
   autoload :Watcher,             "bridgetown-core/watcher"
   autoload :YAMLParser,          "bridgetown-core/yaml_parser"
 
+  FrontmatterDefaults = ActiveSupport::Deprecation::DeprecatedConstantProxy.new(
+    "FrontmatterDefaults",
+    "Bridgetown::FrontMatter::Defaults"
+  )
+
+  FrontMatterImporter = ActiveSupport::Deprecation::DeprecatedConstantProxy.new(
+    "FrontMatterImporter",
+    "Bridgetown::FrontMatter::Importer"
+  )
```

Comment:
**comment:** I kept these to honor the [move fast but try not to break things](https://www.bridgetownrb.com/docs/philosophy#core-principles) principle.

**question:** These don't use the `Bridgetown::Deprecator`. Do we want to refactor to use it?